packages/ballerina-extension/src/utils/project-artifacts.ts

```
@@ -324,11 +325,17 @@ async function getEntryValue(artifact: BaseArtifact, projectPath: string, icon:
             break;
         case DIRECTORY_MAP.LISTENER:
             // Do things related to listener
-            entryValue.icon = getCustomEntryNodeIcon(getTypePrefix(artifact.module));
+            const listenerModule = getTypePrefix(artifact.module);
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)